Still some PEP8 issues:

```
./delivery_carrier_label_postlogistics/__openerp__.py:90:81: E501 line too long (81 > 80 characters)
./delivery_carrier_label_postlogistics/company.py:29:81: E501 line too long (118 > 80 characters)
./delivery_carrier_label_postlogistics/postlogistics/web_service.py:101:81: E501 line too long (82 > 80 characters)
./delivery_carrier_label_dispatch/picking_dispatch.py:89:81: E501 line too long (84 > 80 characters)
./delivery_carrier_label_dispatch/tests/test_generate_labels.py:36:81: E501 line too long (92 > 80 characters)
./delivery_carrier_label_dispatch/tests/test_generate_labels.py:79:81: E501 line too long (101 > 80 characters)
./delivery_carrier_file_laposte/carrier_file.py:29:81: E501 line too long (87 > 80 characters)
```

--- a/delivery_carrier_file_laposte/carrier_file.py
+++ b/delivery_carrier_file_laposte/carrier_file.py
@@ -26,7 +26,8 @@ class carrier_file(osv.osv):
     _inherit = 'delivery.carrier.file'
 
     def get_type_selection(self, cr, uid, context=None):
-        result = super(carrier_file, self).get_type_selection(cr, uid, context=context)
+        result = super(carrier_file, self
+                       ).get_type_selection(cr, uid, context=context)
         if 'la_poste' not in result:
             result.append(('la_poste', 'La Poste'))
         return result

--- a/delivery_carrier_file_laposte/generator/laposte_generator.py
+++ b/delivery_carrier_file_laposte/generator/laposte_generator.py
@@ -21,7 +21,9 @@
 
 import csv
 
-from openerp.addons.base_delivery_carrier_files.generator import CarrierFileGenerator
+from openerp.addons.base_delivery_carrier_files.generator import (
+    CarrierFileGenerator
+    )
 from openerp.addons.base_delivery_carrier_files.generator import BaseLine
 from openerp.addons.base_delivery_carrier_files.csv_writer import UnicodeWriter
 
@@ -57,24 +59,30 @@ class LaPosteFileGenerator(CarrierFileGenerator):
         return carrier_name == 'la_poste'
 
     def _get_filename_single(self, picking, configuration, extension='csv'):
-        return super(LaPosteFileGenerator, self)._get_filename_single(picking, configuration, extension='txt')
+        return super(LaPosteFileGenerator, self
+                     )._get_filename_single(picking, configuration,
+                                            extension='txt')
 
     def _get_filename_grouped(self, configuration, extension='csv'):
-        return super(LaPosteFileGenerator, self)._get_filename_grouped(configuration, extension='txt')
+        return super(LaPosteFileGenerator, self
+                     )._get_filename_grouped(configuration, extension='txt')
 
     def _get_rows(self, picking, configuration):
         """
         Returns the rows to create in the file for a picking
 
-        :param browse_record picking: the picking for which we generate a row in the file
-        :param browse_record configuration: configuration of the file to generate
+        :param browse_record picking: the picking for which we generate a row
+               in the file
+        :param browse_record configuration: configuration of the file to
+               generate
         :return: list of rows
         """
         line = LaPosteLine()
         line.reference = picking.name
         address = picking.address_id
         if address:
-            line.lastname = address.name or (address.partner_id and address.partner_id.name)
+            line.lastname = address.name or (address.partner_id
+                                             and address.partner_id.name)
             # if a company, put company name
             if picking.address_id.partner_id.title:
                 line.company_name = address.partner_id.name
@@ -94,7 +102,8 @@ class LaPosteFileGenerator(CarrierFileGenerator):
 
         :param StringIO file_handle: file to write in
         :param rows: rows to write in the file
-        :param browse_record configuration: configuration of the file to generate
+        :param browse_record configuration: configuration of the file to
+               generate
         :return: the file_handle as StringIO with the rows written in it
         """
         writer = UnicodeWriter(file_handle, delimiter=';', quotechar='"',

--- a/delivery_carrier_label_dispatch/picking_dispatch.py
+++ b/delivery_carrier_label_dispatch/picking_dispatch.py
@@ -86,7 +86,8 @@ class PickingDispatch(Model):
             }
         return res
 
-    def option_ids_change(self, cr, uid, ids, option_ids, carrier_id, context=None):
+    def option_ids_change(self, cr, uid, ids, option_ids, carrier_id,
+                          context=None):
         carrier_obj = self.pool.get('delivery.carrier')
         res = {}
         if not carrier_id:

--- a/delivery_carrier_label_dispatch/tests/test_generate_labels.py
+++ b/delivery_carrier_label_dispatch/tests/test_generate_labels.py
@@ -33,7 +33,8 @@ class test_generate_labels(common.TransactionCase):
         self.Picking = self.registry('stock.picking')
         self.ShippingLabel = self.registry('shipping.label')
         self.PickingDispatch = self.registry('picking.dispatch')
-        self.DeliveryCarrierLabelGenerate = self.registry('delivery.carrier.label.generate')
+        self.DeliveryCarrierLabelGenerate = self.registry(
+            'delivery.carrier.label.generate')
 
         picking_out_1_id = self.Picking.create(
             cr, uid,
@@ -76,7 +77,8 @@ class test_generate_labels(common.TransactionCase):
              })
 
         label = ''
-        dummy_pdf_path = get_module_resource('delivery_carrier_label_dispatch', 'tests', 'dummy.pdf')
+        dummy_pdf_path = get_module_resource('delivery_carrier_label_dispatch',
+                                             'tests', 'dummy.pdf')
         with file(dummy_pdf_path) as dummy_pdf:
             label = dummy_pdf.read()
 

--- a/delivery_carrier_label_postlogistics/__openerp__.py
+++ b/delivery_carrier_label_postlogistics/__openerp__.py
@@ -87,7 +87,8 @@ Technical references
 
 `"Barcode" web service documentation`_
 
-.. _"Barcode" web service documentation: http://www.poste.ch/post-barcode-cug.htm
+.. _"Barcode" web service documentation: http://www.poste.ch/post-barcode-cug\
+.htm
 
 
 Contributors

--- a/delivery_carrier_label_postlogistics/company.py
+++ b/delivery_carrier_label_postlogistics/company.py
@@ -26,7 +26,9 @@ class ResCompany(orm.Model):
     _inherit = 'res.company'
 
     def _get_wsdl_url(self, cr, uid, ids, field_name, arg, context=None):
-        wsdl_file, wsdl_path = file_open('delivery_carrier_label_postlogistics/data/barcode_v2_2.wsdl', pathinfo=True)
+        wsdl_file, wsdl_path = file_open(
+            'delivery_carrier_label_postlogistics/data/barcode_v2_2.wsdl',
+            pathinfo=True)
         wsdl_url = 'file://' + wsdl_path
         res = dict.fromkeys(ids, wsdl_url)
         return res

--- a/delivery_carrier_label_postlogistics/postlogistics/web_service.py
+++ b/delivery_carrier_label_postlogistics/postlogistics/web_service.py
@@ -98,7 +98,8 @@ class PostlogisticsWebService(object):
             lang = company.partner_id.lang
         lang = self._get_language(lang)
         request = self.client.service.ReadAllowedServicesByFrankingLicense
-        return self._send_request(request, FrankingLicense=license, Language=lang)
+        return self._send_request(request, FrankingLicense=license,
+                                  Language=lang)
 
     def read_service_groups(self, company, lang):
         """ Get group of services """

